clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -3818,13 +3899,16 @@ private void gracefulCloseTest(ConsumerCoordinator coordinator, boolean shouldLe
             return commitRequest.data().groupId().equals(groupId);
         }, new OffsetCommitResponse(new OffsetCommitResponseData()));
 
+        // add t1p to the committed offset metadata cache, we'll then check that the cache is invalidated after revocation which happens during close
+        coordinator.committedOffsetsCache().put(t1p, new OffsetAndMetadata(1L));
```

Comment:
I know you've added a test to verify revoke case, but it's an instance close case. Could we add one more revoke test in `testRejoinGroup` for a usual revoke case? Thanks.